src/main/java/com/aws/greengrass/logging/impl/GreengrassLogMessage.java

```diff
@@ -115,7 +123,27 @@ public String getJSONMessage() {
         try {
             return OBJECT_MAPPER.writeValueAsString(this);
         } catch (JsonProcessingException e) {
-            return "{\"error\": \"" + e.getMessage() + "\"}";
+            return "{\"error\": \"" + new String(JsonStringEncoder.getInstance().quoteAsString(e.getMessage())) + "\"}";
```

Comment:
Why `JsonStringEncoder` instead of just using the ObjectMapper that we already have?

Reply:
Seems `ObjectMapper` doesn't have a method to just escape a string. If we use `writeValueAsString` we'll need to handle `JsonProcessingException` again...